### PR TITLE
Add details to renderdoc_frame_capture.md

### PR DESCRIPTION
**Pull request by Hixie**
I still haven't figured out how to make this work, but here's some additional details that I did figure out!

### CODE REVIEW

**Review by chinmaygarde — APPROVED** [review]
I don't think I've ever had to build renderdoc from source. Which environment are you running on?

File: impeller/docs/renderdoc_frame_capture.md [hunk 1]
```diff
@@ -17,15 +20,19 @@
 
    Building a "debug_unopt" build ensures that you have tracing enabled. Without this, RenderDoc will not have much to show.
 
-3. Start RenderDoc, and click the "Launch Application" button.
+3. Start RenderDoc using the `qrenderdoc` command, and (if necessary) select "Launch Application" button from the menu:
```
Comment (loic-sharma) [inline impeller/docs/renderdoc_frame_capture.md:23]: On Windows this is a normal GUI application that isn't launched from the command line. Should we split into two steps. Something like:

```suggestion
3. Start RenderDoc. On Linux, use the `qrenderdoc` command
4. If necessary, select "Launch Application" button ....
```

**Comment by Hixie** [comment]
> I don't think I've ever had to build renderdoc from source. Which environment are you running on?

Some variant of Ubuntu. I've no idea why the version from the package manager was crashing on startup, but the version built from source worked fine.

File: impeller/docs/renderdoc_frame_capture.md (hunk 1 above)
Comment (Hixie) [reply impeller/docs/renderdoc_frame_capture.md]: I tweaked it a bit for ya. :-)

**Pull request merged**